project-reports/2024/2024-Q3-Hyperledger-Cacti.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Cacti
+parent: 2024
+grand_parent: Project Updates
+---
+
+
+# Project Health
+
+- The project is in good health!
+- Multiple mentorship projects are underway, namely [Enabling Enterprise Organizational Blockchain Interoperability](https://wiki.hyperledger.org/display/INTERN/Enabling+Enterprise+Organizational+Blockchain+Interoperability) and [Cacti: Core Operators Modules for DLTs](https://wiki.hyperledger.org/display/INTERN/Cacti%3A+Core+Operators+Modules+for+DLTs).
+- `v2.0.0-rc.2` was released a couple weeks ago and `v2.0.0-rc.3` is not far behind (hopefully next week)!
```

Comment:
```suggestion
- `v2.0.0-rc.2` was released a couple of weeks ago and `v2.0.0-rc.3` is not far behind (hopefully next week)!
```

Reply:
@arsulegai Sorry for the extremely slow response! I would've been fine with accepting this suggestion of course, but now it's too late. Regardless, I wanted to come back and thank you for it and apologize again.